README.md

```diff
@@ -54,7 +54,7 @@ generation [script](./hack/self-signed-ca.sh).
 KFServing can be installed standalone if your kubernetes cluster meets the above prerequisites and KFServing controller is deployed in `kfserving-system` namespace.
 
 ```
-TAG=v0.5.0
+TAG=v0.5.1
 ```
 
 Install KFServing CRD
```

Comment:
In v0.5.1, the crd yaml and controller yaml are now in the same file (`kfserving.yaml`), so the instructions below need to be changed by removing the instruction to apply the `kfserving_crds.yaml` file.

Reply:
Great catch!